src/Visualization/Python/ApplyPointwise.py

```diff
@@ -244,6 +246,11 @@ def apply_pointwise(volfiles: Union[spectre_h5.H5Vol,
         files will be transformed.
       kernels: List of pointwise transformations to apply to the volume data
         in the form of 'Kernel' objects.
+      integrate: Compute the volume integral over the kernels instead of
+        writing them back into the volume files.
```

Comment:
Can you explicitly state what needs to be returned? I don't understand what happens if I return `\gamma_{ij}`. Do I get `\int d\xi J \gamma_{ij}` or `d\xi \gamma_{ij}`? Or can I only return scalars? Do I need to return the tensors multiplied by the Jacobian determinant?

Reply:
When you return `\gamma_{ij}` from the kernel then the integral that's computed is `\int d\xi J \gamma_{ij}`. That's one integral per component in inertial coords. Basically you write a kernel that returns a tensor and then decide "let's integrate it over the volume".